packages/e2e-tests/helpers/pause-information-panel.ts

```diff
@@ -352,7 +352,12 @@ export async function waitForPaused(page: Page, line?: number): Promise<void> {
   }
 }
 
-export async function waitForScopeValue(page: Page, name: string, expectedValue: Expected) {
+export async function waitForScopeValue(
+  page: Page,
+  name: string,
+  expectedValue: Expected,
+  expandScopes = true
```

Comment:
What would it mean to call this function _without_ this value being true? I think it would mean we'd be stuck waiting forever for a value that wasn't in the HTML. Seems bad?